src/main/java/bio/terra/cloudevents/CloudStorageEventType.java

```diff
@@ -0,0 +1,48 @@
+package bio.terra.cloudevents;
+
+/**
+ * For a enumeration of all Cloud Event types, please refer to
+ * https://cloud.google.com/functions/docs/calling/storage
+ */
+public enum CloudStorageEventType {
+  GOOGLE_STORAGE_OBJECT_FINALIZE(
+      "google.storage.object.finalize", "Google Storage Object Finalize"),
+  GOOGLE_STORAGE_OBJECT_ARCHIVE("google.storage.object.archive", "Google Storage Object Archive"),
+  GOOGLE_STORAGE_OBJECT_DELETE("google.storage.object.delete", "Google Storage Object Delete"),
+  GOOGLE_STORAGE_OBJECT_V1_METADATAUPDATED(
+      "google.storage.object.metadataUpdate", "Google Storage Object Metadata Update");
+
+  private String eType;
+  private String eDesc;
+
+  private CloudStorageEventType(String eType, String eDesc) {
```

Comment:
enum constructors are private by default.  
```suggestion
  CloudStorageEventType(String eType, String eDesc) {
```

Reply:
Removed `private` type.